python/metatomic_torch/metatomic/torch/__init__.py

```diff
@@ -50,12 +52,20 @@
 
 
 def __getattr__(name):
-    # lazy import for ase_calculator, making it accessible as
-    # ``metatomic.torch.ase_calculator`` without requiring a separate import from
-    # ``metatomic.torch``, but only importing the code when actually required.
-    if name == "ase_calculator":
-        import metatomic.torch.ase_calculator
-
-        return metatomic.torch.ase_calculator
-    else:
-        raise AttributeError(f"module {__name__!r} has no attribute {name!r}")
+    """
+    Lazily import the `ase_calculator` module or the `MetatomicCalculator`
+    class upon first access.
+    """
+    if name in ("ase_calculator", "MetatomicCalculator"):
+        # Import the submodule once using importlib for a clean, relative import
+        module = importlib.import_module(".ase_calculator", __name__)
+
+        if name == "ase_calculator":
+            globals()[name] = module
+            return module
```

Comment:
I don't get this. `import metatomic.torch.ase_calculator` should already cache modules in `sys.modules`. Why is this needed?